Mathlib/SetTheory/Ordinal/Family.lean

```diff
@@ -919,24 +922,25 @@ theorem blsub_comp {o o' : Ordinal.{max u v}} {f : ∀ a < o, Ordinal.{max u v w
     (fun {_ _} _ _ h => succ_le_succ_iff.2 (hf _ _ h)) g hg
 
 theorem IsNormal.bsup_eq {f : Ordinal.{u} → Ordinal.{max u v}} (H : IsNormal f) {o : Ordinal.{u}}
-    (h : IsLimit o) : (Ordinal.bsup.{_, v} o fun x _ => f x) = f o := by
+    (h : IsSuccLimit o) : (Ordinal.bsup.{_, v} o fun x _ => f x) = f o := by
   rw [← IsNormal.bsup.{u, u, v} H (fun x _ => x) h.ne_bot, bsup_id_limit fun _ ↦ h.succ_lt]
 
 theorem IsNormal.blsub_eq {f : Ordinal.{u} → Ordinal.{max u v}} (H : IsNormal f) {o : Ordinal.{u}}
-    (h : IsLimit o) : (blsub.{_, v} o fun x _ => f x) = f o := by
+    (h : IsSuccLimit o) : (blsub.{_, v} o fun x _ => f x) = f o := by
   rw [← IsNormal.bsup_eq.{u, v} H h, bsup_eq_blsub_of_lt_succ_limit h]
   exact fun a _ => H.1 a
 
 theorem isNormal_iff_lt_succ_and_bsup_eq {f : Ordinal.{u} → Ordinal.{max u v}} :
-    IsNormal f ↔ (∀ a, f a < f (succ a)) ∧ ∀ o, IsLimit o → (bsup.{_, v} o fun x _ => f x) = f o :=
+    IsNormal f ↔ (∀ a, f a < f (succ a)) ∧
+      ∀ o, IsSuccLimit o → (bsup.{_, v} o fun x _ => f x) = f o :=
```

Comment:
Not relevant to this PR, but it might be worth rearranging the universe variables to `bsup` so that the one that needs to be set comes first. Then you can write `bsup.{v}` instead of `bsup.{_, v}`